plugins/scaffolder/src/types.ts

```diff
@@ -78,24 +102,49 @@ export type LogEvent = {
   taskId: string;
 };
 
+/**
+ * The input options to the `scaffold` method of the `ScaffolderClient`.
+ *
+ * @public
+ */
 export interface ScaffolderScaffoldOptions {
   templateRef: string;
   values: Record<string, JsonValue>;
   secrets?: Record<string, string>;
 }
 
+/**
+ * The response shape of the `scaffold` method of the `ScaffolderClient`.
+ *
+ * @public
+ */
 export interface ScaffolderScaffoldResponse {
   taskId: string;
 }
 
+/**
+ * The arguments for `getIntergationsList`.
```

Comment:
```suggestion
 * The arguments for `getIntegrationsList`.
```

Signed-off-by: Fredrik Adelöw <freben@gmail.com>